docs/install/core-install.rst

```diff
@@ -0,0 +1,201 @@
+              install, deploy, Grafana, client, configuration, modulefiles
+
+*********************************
+Installing and deploying Omniperf
+*********************************
+
+Omniperf consists of two installation components.
+
+* :ref:`Omniperf core installation <core-install>` (client-side)
+
+  * Provides the core application profiling capability.
+  * Allows the collection of performance counters, filtering by hardware
+    block, dispatch, kernel, and more.
+  * Provides a CLI-based analysis mode.
+  * Provides a standalone web interface for importing analysis metrics.
+
+* :doc:`Grafana server for Omniperf <grafana-setup>` (server-side) (*optional*)
+
+  * Hosts the MongoDB backend and Grafana instance.
+  * Is packaged in a Docker container for easy setup.
+
+Determine what you need to install based on how you would like to interact with
+Omniperf. See the following decision tree to help determine what installation is
+right for you.
+
+.. image:: ../data/install/install-decision-tree.png
+   :align: center
+   :alt: Decision tree for installing and deploying Omniperf
+
+.. _core-install:
+ 
+Core installation
+=================
+
+The core Omniperf application requires the following basic software
+dependencies. The core Omniperf is included with your ROCm installation.
+
+* Python ``>= 3.8``
+* CMake ``>= 3.19``
+* ROCm ``>= 5.7.1``
+
+Omniperf depends on a number of Python packages documented in the top-level
+``requirements.txt`` file. Install these *before* configuring Omniperf.
+
+.. tip::
+
+   If looking to build Omniperf as a developer, consider these additional
+   requirements.
+
+   .. list-table::
+
+       * - ``docs/sphinx/requirements.txt``
+         - Python packages required to build this documentation from source.
+
+       * - ``requirements-test.txt``
+         - Python packages required to run Omniperf's CI suite using PyTest.
+
+The recommended procedure for Omniperf usage is to install into a shared file
+system so that multiple users can access the final installation. The
+following steps illustrate how to install the necessary Python dependencies
+using `pip <https://packaging.python.org/en/latest/>`_ and Omniperf into a
+shared location controlled by the ``INSTALL_DIR`` environment variable.
+
+.. _core-install-cmake-vars:
+
+Configuration variables
+-----------------------
+The following installation example leverages several
+`CMake <https://cmake.org/cmake/help/latest>`_ project variables defined as
+follows.
+
+.. list-table::
+    :header-rows: 1
+
+    * - CMake variable
+      - Description
+
+    * - ``CMAKE_INSTALL_PREFIX``
+      - Controls the install path for Omniperf files.
+
+    * - ``PYTHON_DEPS``
+      - Specifies an optional path to resolve Python package dependencies.
+
+    * - ``MOD_INSTALL_PATH``
+      - Specifies an optional path for separate Omniperf modulefile installation.
+
+.. _core-install-steps:
+
+Installation steps
+------------------
+
```

Comment:
Hi @peterjunpark , 

What are your thoughts about adding a subsection for "installing via package", which should be possible after rocm-6.2. 
Assuming rocm-6.2 is already installed, it should be something like this: 
```
sudo apt install omniperf
pip install -r /opt/rocm/libexec/omniperf/requirements.txt
```
Replace `apt` with the appropriate package installer for your OS.

Reply:
Thanks @dgaliffiAMD. Added a thing.

Could you review this? https://advanced-micro-devices-demo--362.com.readthedocs.build/projects/omniperf/en/362/install/core-install.html#install-via-package-manager

Let me know if there are any mistakes or if you have any other suggestions. Thanks!